hdmap/hdmap_ontology.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
```

Comment:
the _instance.json file is missing. 
The rest looks good now.

Reply:
add hdmap-instance.json. done!